slsim/lsst_science_pipeline.py

```diff
@@ -749,6 +762,40 @@ def dp0_time_series_images_data(butler, center_coord, radius="0.1", band="i", si
     return table_data
 
 
+def multiple_dp0_time_series_images_data(
+    butler, center_coords_list, radius="0.034", band="i", size=101, output_file=None
+):
+    """Creates multiple time series data from dp0 data.
```

Comment:
can you elaborate a bit what 'multiple' means. Multiple bands? Multiple objects with different cutouts?